NEWS.md

```diff
@@ -1,15 +1,76 @@
-# In progress:
-* Installation of local snap components
-* Started support for snap services to show real status of user daemons
-
-# Next:
-* state: add support for notices (from pebble)
-* daemon: add notices to the snapd API under `/v2/notices` and `/v2/notice`
-* Mandatory device cgroup for all snaps using bare and core24 base as well as future bases
-* Added API route for creating recovery systems: POST to `/v2/systems` with action `create`
-* Added API route for removing recovery systems: POST to `/v2/systems/{label}` with action `remove`
-* Support for user daemons by introducing new control switches --user/--system/--users for service start/stop/restart
-* client,daemon: expose features supported/enabled in `/v2/system-info`
+# New in snapd 2.62:
+* Aspects based configuration schema support (experimental)
+* Refresh app awareness support for UI (experimental)
+* Support for user daemons by introducing new control switches --user/--system/--users for service start/stop/restart (experimental)
+* Support creating and removing recovery systems for use by factory reset
+* Enable API route for creating and removing recovery systems using /v2/systems with action create and /v2/systems/{label} with action remove
+* Lift requirements for fde-setup hook for single boot install
+* Enable single reboot gadget update for UC20+
+* Allow core to be removed on classic systems
+* Support for remodeling on hybrid systems
+* Install desktop files on Ubuntu Core and update after snapd upgrade
+* Upgrade sandbox features to account for cgroup v2 device filtering
+* Support snaps to manage their own cgroups
+* Add support for AppArmor 4.0 unconfined profile mode
+* Add AppArmor based read access to /etc/default/keyboard 
+* Upgrade to squashfuse 0.5.0
+* Support useradd utility to enable removing Perl dependency for UC24+
+* Support for recovery-chooser to use console-conf snap
+* Add support for --uid/--gid using strace-static  
+* Add support for notices (from pebble) and expose via the snapd REST API endpoints /v2/notices and /v2/notice
+* Expose AppArmor prompting experimental features supported/enabled in snapd REST API endpoint /v2/system-info
+* Add polkit authentication for snapd REST API endpoints /v2/snaps/{snap}/conf and /v2/apps
+* Add refresh-inhibit-proceed-time field to snapd REST API endpoint /v2/snaps
+* Take into account validation sets during remodeling
+* Improve offline remodeling to use installed revisions of snaps to fulfill the remodel revision requirement
+* Add rpi configuration option sdtv_mode
+* When snapd snap is not installed, pin policy ABI to 4.0 or 3.0 if present on host
+* Fix gadget zero-sized disk mapping caused by not ignoring zero sized storage traits
+* Fix gadget install case where size of existing partition was not correctly taken into account
+* Fix trying to unmount early kernel if none
```

Comment:
I don't quite understand if this is missing a comma or if this makes sense as a sentence? What is none here? Is it the kernel?

Reply:
This relates to a check whether a path exists before we unmount it - but yes could be formuated clearer

Reply:
Will fix

Reply:
Fixed